completion_aggregator/models.py

```diff
@@ -171,8 +172,44 @@ def submit_completion(self, user, course_key, block_key, aggregation_name, earne
                 'last_modified': last_modified,
             },
         )
+        self.emit_completion_aggregator_logs([obj])
+
         return obj, is_new
 
+    @staticmethod
+    def emit_completion_aggregator_logs(updated_aggregators):
+        """
+        Emit a tracking log for each element of the list parameter.
+
+        Parameters
+        ----------
+            updated_aggregators: List of Aggregator intances
+
+        """
+        for obj in updated_aggregators:
```

Comment:
Why you change to  `obj` if in other places is `agregator`.
https://github.com/nelc/openedx-completion-aggregator/blob/ead8f61b1081a87aa7e37887d82724f807320129/completion_aggregator/models.py#L221

Reply:
I understand your concern and IMO it's much better  `aggregator ` however your argument is not valid since in other places is obj 

https://github.com/nelc/openedx-completion-aggregator/blob/ead8f61b1081a87aa7e37887d82724f807320129/completion_aggregator/models.py#L163

https://github.com/nelc/openedx-completion-aggregator/blob/ead8f61b1081a87aa7e37887d82724f807320129/completion_aggregator/models.py#L232